fastly/secret_store_test.go

```diff
@@ -171,6 +176,79 @@ func TestClient_CreateSecret(t *testing.T) {
 	}
 }
 
+func TestClient_CreateSecret_clientEncryption(t *testing.T) {
+	t.Parallel()
+
+	ss := createSecretStoreHelper(t, 0)
+
+	var (
+		ck  *ClientKey
+		err error
+	)
+
+	record(t, fmt.Sprintf("secret_store/%s/create_client_key", t.Name()), func(c *Client) {
+		ck, err = c.CreateClientKey()
+	})
+	if err != nil {
+		t.Fatalf("error creating client key: %v", err)
+	}
+
+	if got := ck.PublicKey; len(got) == 0 {
+		t.Errorf("PublicKey: got empty")
+	}
+
+	if got := ck.Signature; len(got) == 0 {
+		t.Errorf("Signature: got empty")
+	}
+
+	if got, now := ck.ExpiresAt, time.Now(); got.Before(now) {
```

Comment:
Will there be issues using `time.Now()` considering we're producing a static fixture?

Reply:
I was looking in the codebase for other similar checks and didn't find any.  It doesn't look like the date is encoded in the test fixtures at all (other than a `Date` response header).  Would it be ok to hardcode a date in the test (resetting it to now if `vcrDisabled` is set)?  Or should I loosen this check to just seeing if the value is present?  Or drop the check altogether?

Reply:
Actually, never mind.  I think I can use the server's Date response header to calibrate things.

Reply:
Er, no, ignore that last comment.  I don't have access to the response headers at the time I'm checking.

Reply:
I would maybe just avoid this check for now unless we have a good way to handle it for vcr type fixtures.